README.md

```diff
@@ -14,6 +14,7 @@ Note : 2019 conferences are listed [here](year-pages/2019-conferences-list.md).
 | [Devops Days India](http://devopsdaysindia.org/) | 8-9 Dec | Sheraton Grand, Bengaluru | Devopsdays is a worldwide series of technical conferences covering topics of software development, IT infrastructure operations, and the intersection between them. | 
 | [JSFoo](https://jsfoo.in/2018/) | 26-27 Oct | NIMHANS Convention Centre, Bengaluru | The eighth edition of India’s first JavaScript conference.|
 | [PyCon India](https://in.pycon.org/2018/) | 05-09 Oct | Hyderabad International Convention Centre (HICC), Hyderabad | The premier conference in India on using and developing the Python programming language. |
+| [AWS Community Day Bengaluru 2018](https://communityday.awsugblr.in/) | 05-06 Oct | The Ritz-Carlton, Bangalore | Use Cases / Success Stories and Workshops on Amazon Web Services |
```

Comment:
Can you remove the "2018" in the name? The list is for 2018 should without it would be fine.

Reply:
sure.